assets/src/scss/blocks/core-overrides/Heading.scss

```diff
@@ -1,18 +1,24 @@
-.is-style-chevron::after {
-  content: "";
-  pointer-events: none;
-  margin-inline-start: $sp-1;
-  height: 12px;
-  width: 12px;
-  display: inline-block;
-  mask-image: url("../../public/images/icons/chevron.svg");
-  mask-repeat: no-repeat;
-  mask-size: contain;
-  background-repeat: no-repeat;
-  background-color: currentColor;
-  vertical-align: middle;
+.wp-block-heading {
+  &.is-style-chevron::after {
+    content: "";
+    pointer-events: none;
+    margin-inline-start: $sp-1;
+    height: 12px;
+    width: 12px;
+    display: inline-block;
+    mask-image: url("../../public/images/icons/chevron.svg");
```

Comment:
You need to update this URL, it no longer works

Reply:
Done!